doc(release) Release notes modules 24.10 (#4298)
centreon/centreon-documentation: versioned_docs/version-24.10/releases/centreon-commercial-extensions.mdx

## Centreon MAP

### 24.10.5

Release date: `April 30, 2025`

<details>
  <summary>Bug fixes</summary>

- [Editor] Added missing macros to the Output widget.

> - [Editor] Fixed issue preventing service groups from being available in the bulk resource selection.

Reviewer: ```suggestion
- [Editor] Fixed an issue preventing service groups from being available in the bulk resource selection.
```

- [Editor] Removed deleted service relations on the detailed view of a host.

> - [Geoviews] Fixed a display issue preventing a large list of hostgroups from being displayed correctly when selecting the resources to add.

Reviewer: ```suggestion
- [Geoviews] Fixed a display issue preventing a large list of host groups from being displayed correctly when selecting the resources to add.
```

- [Server] Fixed an issue that could duplicate unwanted elements.

> - [Server] Fixed an issue that may prevent user from duplicating a map.

Reviewer: ```suggestion
- [Server] Fixed an issue that could prevent users from duplicating a map.
```

## Centreon BAM

### 24.10.4

Release date: `April 30, 2025`

<details open>
  <summary>Enhancements</summary>

> - [Dashboards] BA Diagram widget - Added the parent host name on service KPI.

Reviewer: ```suggestion
- [Dashboards] BA Diagram widget - Added the parent host name on service KPIs.
```

</details>

<details>
  <summary>Bug fixes</summary>

> - [BAM] Fixed issue preventing BA from being added to a remote server if one of its KPI was a BA itself.

Reviewer: ```suggestion
- [BAM] Fixed issue preventing BAs from being added to a remote server if one of their KPIs was a BA itself.
```

## Centreon MBI

### 24.10.3

Release date: `April 30, 2025`

<details>
  <summary>Bug fixes</summary>

> - [MBI] Fixed issue preventing reports from being exported in XLSX format.

Reviewer: ```suggestion
- [MBI] Fixed an issue preventing reports from being exported in XLSX format.
```

## Centreon Auto Discovery

### 24.10.4

Release date: `April 30, 2025`

<details>
  <summary>Bug fixes</summary>

> - [Discovery] Fixed an issue where manually scanned services could not be saved by users with ACL.

Reviewer: ```suggestion
- [Discovery] Fixed an issue where manually scanned services could not be saved by non-admin users.
```
